stock_picking_partner_ref/pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["whool"]
+build-backend = "whool.buildapi"
```

```suggestion
[build-system]
requires = ["whool"]
build-backend = "whool.buildapi"
```

This file can be deleted.

---

Is automatically generated with the pre-commit

---

You are right